Formula/libetpan.rb

```diff
@@ -0,0 +1,23 @@
+
+class Libetpan < Formula
+  desc "Mail library is to provide a portable, efficient framework."
```

Comment:
No period at the end of `desc`.
The sentence should be reworked, maybe _“Portable mail library handling several protocols”_?